WordPress/Classes/ViewRelated/Cells/InlineEditableNameValueCell.swift

```diff
@@ -61,19 +65,33 @@ class InlineEditableNameValueCell: WPTableViewCell, NibReusable {
 
     @objc func textFieldDidChange(textField: UITextField) {
         textField.text = textField.text?.replacingOccurrences(of: " ", with: "\u{00a0}")
-        delegate?.inlineEditableNameValueCell?(self, valueTextFieldDidChange: textField)
+
+        let text = sanitizedText(for: textField)
+        delegate?.inlineEditableNameValueCell?(self, valueTextFieldDidChange: text)
     }
 
     @objc func textEditingDidEnd(textField: UITextField) {
-        textField.text = textField.text?.replacingOccurrences(of: "\u{00a0}", with: " ")
-        delegate?.inlineEditableNameValueCell?(self, valueTextFieldEditingDidEnd: textField)
+        let text = sanitizedText(for: textField)
+
+        textField.text = text
+        delegate?.inlineEditableNameValueCell?(self, valueTextFieldEditingDidEnd: text)
+    }
+
+    private func sanitizedText(for textField: UITextField) -> String {
+        return textField.text?.replacingOccurrences(of: "\u{00a0}", with: " ") ?? ""
```

Comment:
Because the `\u{00a0}` is used even above, can we wrap it in a _Constant_ enum?

Reply:
Good idea!